analysis/codelists.py

```diff
@@ -866,183 +866,154 @@
 # Dementia
 # Alzheimer's disease
 alzheimer_snomed = codelist_from_csv(
-    "codelists/bristol-alzheimers-disease-snomed-ct.csv",
+    "codelists/bristol-alzheimers-disease-snomed-ct-v13.csv",
     system = "snomed",
     column = "code",
 )
 alzheimer_icd10 = codelist_from_csv(
-    "codelists/bristol-alzheimers-disease.csv",
+    "codelists/bristol-alzheimers-disease-icd10-v13.csv",
+    system = "icd10",
+    column = "code",
+)
+
+# Any dementia
+any_dementia_snomed = codelist_from_csv(
+    "codelists/bristol-any-dementia-snomed-ct-v14.csv",
+    system = "icd10",
+    column = "code",
```

Comment:
the name looks like ctv (snomed) but the system is icd10, is this correct?

Reply:
It should be system = snomed. I will fix it.